roles/kubernetes-apps/ingress_controller/cert_manager/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+cert_manager_controller_image_repo: quay.io/jetstack/cert-manager-controller
```

Comment:
This should go in the downloads role

Reply:
We did something similar here (at least for those with my contribution):
* https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes-apps/ingress_controller/ingress_nginx/defaults/main.yml#L6
* https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes-apps/external_provisioner/cephfs_provisioner/defaults/main.yml#L3
* https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes-apps/external_provisioner/local_volume_provisioner/defaults/main.yml#L3
* https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes-apps/registry/defaults/main.yml#L3

From my point of view, because they are hosted under [roles/kubernetes-apps](https://github.com/kubernetes-incubator/kubespray/tree/master/roles/kubernetes-apps) so not a MUST for installation, keep it away from [roles/download](https://github.com/kubernetes-incubator/kubespray/tree/master/roles/download) could keep the core architecture of Kubespray clean and elegant.

Reply:
Good point

Reply:
A bit over design for me...

Now if cert_manager_enabled: false, no download image, no install

If true, both download and install will happening together

Moreover, when maintenancing this addon, most likely we just need to focus on its folder, and don’t need to search all around to find if any else missing :-)